internal/pkg/describe/status.go

```diff
@@ -195,12 +195,14 @@ func (s *ServiceStatusDesc) HumanString() string {
 	fmt.Fprint(writer, color.Bold.Sprint("\nTask Status\n\n"))
 	writer.Flush()
 	fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\t%s\t%s\n", "ID", "Image Digest", "Last Status", "Started At", "Stopped At", "Health Status")
+	fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\t%s\t%s\n", "--", "------------", "-----------", "----------", "----------", "-------------")
```

Comment:
can we [strings.Repeat](https://golang.org/pkg/strings/#Repeat) to simplify our lives and possibly less prone to human error?
`strings.repeat("-", len("ID"))`

Tests can remain as is (manual dashes)

Reply:
yes! done.